lua/starfall/libs_cl/bass.lua

```diff
@@ -262,9 +262,9 @@ function bass_methods:getTime ()
 	end
 end
 
---- Gets the FFT of a sound
--- @param n Sample size of the hamming window. Must be power of 2
--- @return FFT table of the sound
+--- Perform fast Fourier transform algorithm to compute the DFT of the sound channel.
+-- @param n Number of consecutive audio samples. Must be in 0-8 range, depending on this argument you will get 256*2^n samples.
```

Comment:
Number of samples are in 0-7 range http://wiki.garrysmod.com/page/Enums/FFT.

Reply:
Thanks, just corrected this one